public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -39,6 +39,35 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 	</div>
 </div>
 
+<b>Connection Certificate Files</b>
+
+<div class="gf-form-group">
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">SSL Root Cert</span>
+ 		<input type="text" class="gf-form-input gf-form-input--has-help-icon" ng-model='ctrl.current.jsonData.sslrootcertfile' placeholder="SSL root cert file"></input>
+		<info-popover mode="right-absolute">
+			If selected SSL Mode needs a SSL Root Certificate, provide here the path to the file. 
+      Be sure that the file is readable by grafana user.
+		</info-popover>
+	</div>
+	<div class="gf-form max-width-30">
+		<span class="gf-form-label width-7">SSL User Cert</span>
+ 		<input type="text" class="gf-form-input gf-form-input--has-help-icon" ng-model='ctrl.current.jsonData.sslcertfile' placeholder="SSL user cert file"></input>
+		<info-popover mode="right-absolute">
+			If selected SSL Mode needs a SSL User Certificate, provide here the path to the file. 
+      Be sure that the file is readable by grafana user.
```

Comment:
```suggestion
      Be sure that the file is readable by the grafana user.
```

Reply:
Remember to capitalize "Grafana."

Reply:
@oddlittlebird Does "grafana" here refer to a specific Unix user? If so, it should probably be lowercase.

Reply:
If we are using Grafana (software or company) as an adjective, which is what it looks like here, then it should be capitalized. It is a proper noun.

If it is something in code where the code would be incorrect and possibly not work if the term is capitalized, then it should be lowercase.

Reply:
@oddlittlebird I'm not 100% sure, but it might be referring to a specific Unix user, in which case I'm pretty sure it should be lowercase.

Reply:
@aknuds1, why? How does the OS in use render "Grafana" not a proper noun?

Reply:
If we are referring to a specific OS user running Grafana, we have to use the same case or there will be a technical mismatch.

Reply:
So, we are referring to `grafana user` in the code sense? In that case, I suggest actually changing it to be a code block to avoid confusion.

Both of these would be correct in my opinion:

* Be sure that the file is readable for `grafana user`.
* Be sure that the file is readable for the Grafana user.

A better solution might be to get even more specific, since I think you are talking about permissions.

Be sure that the `grafana user` has read permissions for this file.

Reply:
@oddlittlebird I'm trying to find out if the sentence is referring to a particular username ("grafana") or simply the user running Grafana. The way it reads currently, it's ambiguous.

Reply:
I agree that it is ambiguous, but I think "the Grafana user" is clear if we are actually referring to a human user. 
 
I dislike the passiveness of "the user running Grafana."

If we are referring to some part of the Grafana program called `grafana user`, then let's specify it by name and wrap it in a code block so that people like me don't look at it and go "Oh, they forgot to capitalize a proper noun!"

Reply:
It's not a part of the program called `grafana user`, just to make that clear. It's referring to one of two:

1. An OS username, f.ex. "grafana"
2. The OS user running Grafana (which could have the username "grafana" or something else)

Reply:
How about Grafana OS user?

Reply:
imo "Grafana user" implies user _of_ grafana, and is also technically incorrect.

"`grafana` user" is more correct, in referring to a literal system user, however makes the assumption that the process is executed by the `grafana` user.

"the user executing the `grafana` process" is unambiguous and again refers to a literal process name, not Grafana the org, and the latest commit has made this change.

Reply:
Huzzah, a solution we can all agree on!

the user executing the `grafana` process